pom.xml

```diff
@@ -66,8 +66,8 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.2.0</cdap.version>
-    <slf4j.version>1.7.5</slf4j.version>
+    <cdap.version>6.9.0-SNAPSHOT</cdap.version>
```

Comment:
We may need to maintain support for older versions. Any chance we can set this to 6.6?

Reply:
Logback & Log4j changes are there in cdap 6.8, so least we can use is `6.8`. Do we not have separate release branches in this repo?